src/components/tools/table-tool/use-column-resize.ts

```diff
@@ -1,23 +1,35 @@
 import React, { useCallback } from "react";
-import { TColumn } from "./table-types";
+import { kMinColumnWidth, TColumn } from "./table-types";
+import { TableContentModelType } from "../../../models/tools/table/table-content";
 
 interface IUseColumnResize {
   columns: TColumn[];
-  userColumnWidths: React.MutableRefObject<Record<string, number>>;
+  content: TableContentModelType;
   requestRowHeight: () => void;
+  resizeColumn: React.MutableRefObject<string | undefined>;
+  resizeColumnWidth: React.MutableRefObject<number | undefined>;
   triggerRowChange: () => void;
 }
 export const useColumnResize = ({
-  columns, userColumnWidths, requestRowHeight, triggerRowChange
+  columns, content, requestRowHeight, resizeColumn, resizeColumnWidth, triggerRowChange
 }: IUseColumnResize) => {
-
-  // This is called constantly as the user resizes the column. When we start saving the width to the model,
-  // we'll only want to do so when the user ends adjusting the width.
-  const onColumnResize = useCallback((idx: number, width: number) => {
-    userColumnWidths.current[columns[idx].key] = width;
+  const onColumnResize = useCallback((idx: number, width: number, complete: boolean) => {
```

Comment:
Seems like a comment indicating that this is a non-standard use of RDG that requires our custom modifications would be a good idea.